stories/Components/UIcomponents/Cards/FeaturedCards/featuredContentCard.scss

```diff
@@ -13,6 +13,29 @@
     display: flex;
   }
 
+  @include devicebreak(medium) {
+    .cta__link.cta--arrow {
+      width: 100%;
```

Comment:
@pallavikadam-srijan Kindly change to `rem` based units. Also in the below lines.